modules/avoindata-drupal-theme/less/overrides.less

```diff
@@ -841,6 +841,10 @@ p:last-child,
     font-weight: 700;
   }
 
+  a {
+    color: #ebe7e7;
```

Comment:
Should be a LESS variable?

Reply:
I changed this and many other similar cases to variables. I still left some hard coded values as well in places where they were used only once and there was not a suitable variable defined already.